src/test/java/backend/backendTest.java

```diff
@@ -69,14 +65,18 @@ public static void main(String[] args) {
 
         //designate the last column as the data's 'class'
         data.setClassIndex(data.numAttributes() - 1);
+
         //give data to the WekaTaskManager
-        mgr.setData(data);
+        //mgr.setData(data);
+        dispatch.setData(data);
+        dispatch.setTimeLimit(1);
+        dispatch.launch();
 
         //run some tests on the data.
-        taskList.addTask(lr); // run LR
-        taskList.addTask(nn); // run NN
-        taskList.doTask();
-        taskList.doTask();
+        //taskList.addTask(lr); // run LR
```

Comment:
commented code is commented code, regardless of it being in a test. it leaves an unclear trail of lying bread crumbs